### DISCUSSION TITLE
Libre 1/Miaomiao 2 inaccuracy then failure

### DISCUSSION THREAD

**Discussion by btdkomplex** [NightscoutFoundation/xDrip#2364] [Q&A]
So this started happening a couple of months ago. After a few days with absolutely no issues the BG reading in xDrip would suddenly be completely inaccurate. Sometimes as much as 10mmo/l! This continues until I reset the calibrations which then stops all readings being taken all together. The logs complain that there is too much data.
However sometimes after manually entering in a finger prick test it starts being able to read from the sensor again. If this doesn't happen I end up having to use a fresh sensor, which inevitably fails again a few days later the exact same way.

Has anyone else experienced this? It's costing me a small fortune in Libre sensors :(

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Can you confirm you only calibrate at stable BG when in range? First calibration after resetting should be done at stable BG in the lower normal range.

**Reply by btdkomplex** [reply]
> Can you confirm you only calibrate at stable BG when in range? First calibration after resetting should be done at stable BG in the lower normal range.

Can't speak to the other times but it's just happened again and my bg level is stable. xDrip was reporting 4.1mmol when in reality it was 14.4! The sensor has been running fine for 7 days.

**Reply by psonnera** [reply]
@btdkomplex please enable calibration data table and post it here. https://xdrip.readthedocs.io/en/latest/use/lesscommon/#show-data-tables

**Reply by btdkomplex** [reply]
> @btdkomplex please enable calibration data table and post it here. https://xdrip.readthedocs.io/en/latest/use/lesscommon/#show-data-tables

![Screenshot_20221011-175823](https://user-images.githubusercontent.com/648797/195154244-c786e35c-0114-4500-87cc-d52e5f1a767d.png)

**Reply by psonnera** [reply]
@btdkomplex your sensor is dead. It reads low values when you're high. You can't calibrate it.

**Reply by btdkomplex** [reply]
> @btdkomplex your sensor is dead. It reads low values when you're high. You can't calibrate it.

Are Abbot having manufacturing issues? Don't expect you to know this lol
This happens so often now it's ridiculous. I can't be the only one having this issue surely?

**Reply by psonnera** [reply]
@btdkomplex it really depends on your skin. Make sure you clean it with soap first, dry it and clean it with alcohol. Consider using glue like SkinTack and then cover the sensor with a breathing band-aid like kinesiologic tape. This is what I use on my son and we usually have good results over 2 weeks even during summertime. Some people like holders too.
